project-canvas/src/components/backlog-view/Issue/issueFetcher.ts

```diff
@@ -0,0 +1,55 @@
+import { Issue, Sprint } from "project-extender"
```

Comment:
This file should not exist any more. Please add your changes to the refactored methods and don't forget to updare the useQuery in backlogview